x/gravity/migrations/v1/migrate.go

```diff
@@ -0,0 +1,30 @@
+package v1
+
+import (
+	"cosmossdk.io/core/store"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/palomachain/paloma/x/gravity/exported"
+	"github.com/palomachain/paloma/x/gravity/types"
+)
+
+// migrateParams will set the params to store from subspace
+
+func MigrateParams(
```

Comment:
Have you manually tested if the migration function is being executed? I don't see a consensus version bump for this migration logic in the `module.go` file.

Reply:
No,i didn't test manually.But i now have added the consensus version bump and checked it